Composer/packages/client/src/components/Notifications/NotificationCard.tsx

```diff
@@ -36,7 +37,7 @@ const cardContainer = (show: boolean, ref?: HTMLDivElement | null) => () => {
     border-left: 4px solid #0078d4;
     background: white;
     box-shadow: 0 6.4px 14.4px 0 rgba(0, 0, 0, 0.132), 0 1.2px 3.6px 0 rgba(0, 0, 0, 0.108);
-    min-width: 340px;
+    width: 340px;
```

Comment:
Why are you changing this from min-width to width?

Reply:
The design calls for the card to stay the same width while the text wraps onto more lines. This also means our slide-out Notifications tray doesn't have to change size.